### DISCUSSION TITLE
Add setting for default latency for the live timing

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#231] [Feature requests] [answered]
Add an option in the settings for the default latency for live timing so that it doesn't have to be adjusted each time.

**Comment by jeppeolesen** [comment, marked as answer]
There are options to set the target latency of playbacks, and the updated sync API will match the live timing to it.

![Screenshot 2025-03-23 at 02 01 30](https://github.com/user-attachments/assets/8b35459d-1d04-4e08-8403-986a049e1493)
